README.md

```diff
@@ -47,10 +48,30 @@ Total: 10, Dialing: 1, Established: 7, Closed: 0, Error: 0, NotInitiated: 2
 Total: 10, Dialing: 1, Established: 8, Closed: 0, Error: 0, NotInitiated: 1
 Total: 10, Dialing: 1, Established: 9, Closed: 0, Error: 0, NotInitiated: 0
 Total: 10, Dialing: 0, Established: 10, Closed: 0, Error: 0, NotInitiated: 0
+--- www.google.es:80 tcp test statistics ---
```

Comment:
the command says `--host myhttpsamplehost.com`but the output says `www.google.es:80` 🤔

Reply:
also in the bad output

Reply:
fixed! i didnt want to expose the host i was using for the tests :smile: